app/ui/src/app/integration/edit-page/action-configure/action-configure.component.html

```diff
@@ -64,12 +64,12 @@ <h3>{{ action?.name }}</h3>
       </div>
       <syndesis-loading [loading]="loading">
         <ng-container *ngIf="error">
-          <div [class]="error.class"
-               [innerHtml]="error.message">
-          </div>
-          <div *ngIf="!hasConfiguration" class="control-buttons">
```

Comment:
Removing this `*ngIf` means the buttons will show up in all cases, which we don't actually want, in this case the buttons should only show up if there's no configuration.

Reply:
@gashcrumb sorry about that, added it back.